examples/all-clusters-app/esp32/main/DeviceCallbacks.cpp

```diff
@@ -174,7 +175,7 @@ void DeviceCallbacks::OnIdentifyPostAttributeChangeCallback(EndpointId endpointI
     return;
 }
 
-bool emberAfBasicClusterMfgSpecificPingCallback(void)
+bool emberAfBasicClusterMfgSpecificPingCallback(chip::app::Command *)
```

Comment:
Can we add a default name, for this one and for all the other occurrences where there is no default name.